src/components/NewLocationPage/SummaryStat/utils.ts

```diff
@@ -20,9 +20,9 @@ export const metricSubLabelText: { [key in Metric]: string[] } = {
   [Metric.HOSPITAL_USAGE]: [''],
   [Metric.POSITIVE_TESTS]: [''],
 
-  // TODO(8.2)
+  // TODO(8.2): verify these
```

Comment:
FWIW- I think these look correct now and we could probably remove the TODO.  I assume if they're not correct, Brett / Mikayla / somebody will flag it.